modules/system/classes/asset/Mix.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace System\Classes\Asset;
+
+use Cms\Classes\Theme;
+use Illuminate\Support\HtmlString;
+use Winter\Storm\Support\Facades\Config;
+use Winter\Storm\Support\Facades\Url;
+use Winter\Storm\Support\Str;
+
+class Mix
+{
+    public static function mix(string $path, string $manifestDirectory = ''): HtmlString|string
+    {
+        static $manifests = [];
+
+        $theme = Theme::getActiveTheme();
+
+        $path = Str::start($path, '/');
+
+        if ($manifestDirectory && ! str_starts_with($manifestDirectory, '/')) {
+            $manifestDirectory = "/$manifestDirectory";
+        } else {
+            $manifestDirectory = Str::start($theme->getConfigValue('mix_manifest_path', '/'), '/');
+        }
+
+        $manifestPath = $theme->getPath($theme->getDirName() . '/' . $manifestDirectory . '/mix-manifest.json');
+
+        if (! isset($manifests[$manifestPath])) {
+            if (! is_file($manifestPath)) {
+                throw new \Exception('The Mix manifest does not exist.');
+            }
+
+            $manifests[$manifestPath] = json_decode(file_get_contents($manifestPath), true);
+        }
+
+        $manifest = $manifests[$manifestPath];
+
+        if (! isset($manifest[$path])) {
```

Comment:
```suggestion
        if (!isset($manifest[$path])) {
```
We don't have an auto-styling rule for this yet, but please use this style instead.